src/pages/iou/SplitBillDetailsPage.js

```diff
@@ -0,0 +1,155 @@
+import reportActionPropTypes from '../home/report/reportActionPropTypes';
+import reportPropTypes from '../reportPropTypes';
+import withReportOrNotFound from '../home/report/withReportOrNotFound';
+import FullPageNotFoundView from '../../components/BlockingViews/FullPageNotFoundView';
+import CONST from '../../CONST';
+
+const propTypes = {
+    /* Onyx Props */
+
+    /** The personal details of the person who is logged in */
+    personalDetails: personalDetailsPropType,
+
+    /** The active report */
+    report: reportPropTypes.isRequired,
+
+    /** Array of report actions for this report */
+    reportActions: PropTypes.shape(reportActionPropTypes),
+
+    /** Route params */
+    route: PropTypes.shape({
+        params: PropTypes.shape({
+            /** Report ID passed via route r/:reportID/split/details */
+            reportID: PropTypes.string,
+
+            /** ReportActionID passed via route r/split/:reportActionID */
+            reportActionID: PropTypes.string,
+        }),
+    }).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    personalDetails: {},
+    reportActions: {},
+};
+
+/**
+ * Get the reportID for the associated chatReport
+ *
+ * @param {Object} route
+ * @param {Object} route.params
+ * @param {String} route.params.reportID
+ * @returns {String}
+ */
+function getReportID(route) {
+    return route.params.reportID.toString();
+}
+
+/**
+ * Returns all the participants in the active report
+ *
+ * @param {Object} reportAction The IOU split reportAction, which contains the participants
+ * @param {Object} personalDetails The personal details of the users
+ * @return {Array}
+ */
+const getAllParticipants = (reportAction, personalDetails) => {
+    const participants = lodashGet(reportAction, 'originalMessage.participants', []);
+    const participantsExcludingOwner = _.filter(participants, (participant) => participant !== reportAction.actorEmail);
+
+    return _.chain(participantsExcludingOwner)
+        .map((login) => {
+            const userLogin = Str.removeSMSDomain(login);
+            const userPersonalDetail = lodashGet(personalDetails, login, {displayName: userLogin, avatar: ''});
+
+            return {
+                alternateText: userLogin,
+                displayName: userPersonalDetail.displayName,
+                icons: [
+                    {
+                        source: ReportUtils.getAvatar(userPersonalDetail.avatar, login),
+                        name: login,
+                        type: CONST.ICON_TYPE_AVATAR,
+                    },
+                ],
+                keyForList: userLogin,
+                login,
+                text: userPersonalDetail.displayName,
+                tooltipText: userLogin,
+                participantsList: [{login, displayName: userPersonalDetail.displayName}],
+            };
+        })
+        .sortBy((participant) => participant.displayName.toLowerCase())
+        .value();
+};
```

Comment:
Seems like this is all redundant, will switch to [this method](https://github.com/Expensify/App/pull/19196/files#diff-2eec74c37754b999e449de1d16759a3d34a02b5d9e760184c791ba59ca357080R32-R33) tomorrow

Reply:
Done